src/components/organisms/UiMenu/UiMenu.vue

```diff
@@ -62,125 +50,170 @@ export interface MenuProps {
    */
   items?: (string | MenuRenderItem)[];
   /**
-   * Use this props to allow using arrows to navigation.
+   * Use this props to allow using arrows instead tabs for navigation UiMenuItems.
    */
   enableKeyboardNavigation?: boolean;
+  /**
+   * Use this props to pass refs to UiMenuItems.
+   */
+  itemsTemplateRefs?: any;
```

Comment:
BREAKING CHANGES: pass refs as props instead pushed it on `onMounted` hook from UiMenuItems